### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

**Comment by zhenlineo** [comment]
Hello,

Thanks very much for contributing. Would you mind to sign our CLA https://neo4j.com/developer/cla/ to be able to contribute us?

Also could you smash all commits into one commit?

Thanks again.
Zhen

**Comment by jsoref** [comment]
If you're ok w/ the commits, I'm happy to squash (probably Sunday).

I prefer to leave things as individual commits in case someone objects to a given correction / to enable rebasing.

File: Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApi.feature [hunk 1]
```diff
@@ -86,7 +86,7 @@ Feature: Tests the uniform API of the driver
     Then requesting the `Statement` as text should give: CREATE (:label {name: "Pelle"} )
     And requesting the `Statement` parameter should give: {}
 
-  Scenario: Access Statement parametrised
+  Scenario: Access Statement parameterised
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApi.feature]: Please ignore any spelling errors in path `Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK`
They are auto generated.

File: Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApi.feature.cs [hunk 2]
```diff
@@ -181,10 +181,10 @@ public virtual void AccessStatement()
         
         [Xunit.FactAttribute()]
         [Xunit.TraitAttribute("FeatureTitle", "Tests the uniform API of the driver")]
-        [Xunit.TraitAttribute("Description", "Access Statement parametrised")]
-        public virtual void AccessStatementParametrised()
+        [Xunit.TraitAttribute("Description", "Access Statement parameterised")]
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApi.feature.cs]: Same for this file in path `Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK`

File: Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApiSteps.cs [hunk 3]
```diff
@@ -129,7 +129,7 @@ public void ThenRequestingTheStatementTypeShouldGive(string expected)
             }
             else
             {
-                throw new InvalidOperationException($"Cannot understand the type {expected} in the feature file, while the acutal type returned from the server is {actual}"); 
+                throw new InvalidOperationException($"Cannot understand the type {expected} in the feature file, while the actual type returned from the server is {actual}");
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApiSteps.cs]: Same for this file in path `Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK`

File: Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverStatementResultNavigation.feature [hunk 4]
```diff
@@ -120,14 +120,14 @@ Feature: Statement Result Navigation
       | (:A {"value": 1}) |
       | (:B {"value": 2}) |
 
-  Scenario: Record should contain key and value and value should be retriveble by index
+  Scenario: Record should contain key and value and value should be retrievable by index
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverStatementResultNavigation.feature]: Same for this file in path Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK

File: Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverStatementResultNavigation.feature.cs [hunk 5]
```diff
@@ -467,10 +467,10 @@ public virtual void RecordShouldContainValue()
         
         [Xunit.FactAttribute()]
         [Xunit.TraitAttribute("FeatureTitle", "Statement Result Navigation")]
-        [Xunit.TraitAttribute("Description", "Record should contain key and value and value should be retriveble by index")]
-        public virtual void RecordShouldContainKeyAndValueAndValueShouldBeRetrivebleByIndex()
+        [Xunit.TraitAttribute("Description", "Record should contain key and value and value should be retrievable by index")]
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverStatementResultNavigation.feature.cs]: Same for this file in path Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK

File: Neo4j.Driver/Neo4j.Driver/V1/Config.cs [hunk 6]
```diff
@@ -134,7 +134,7 @@ static Config()
         public EncryptionLevel EncryptionLevel { get; set; } = EncryptionLevel.Encrypted;
 
         /// <summary>
-        /// Gets or sets how to determine the authenticity of an encryption certificate provided by the Neo4j instance we are connecting to.
+        /// Gets or sets how to determine the authenticity of an encryption certificate provided by the Neo4j instance we are connectiong to.
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver/V1/Config.cs]: This was not an error.

File: Neo4j.Driver/Neo4j.Driver/V1/GraphDatabase.cs [hunk 7]
```diff
@@ -39,7 +39,7 @@ public static class GraphDatabase
         ///     If <c>port</c> is not supplied the default of <c>7687</c> will
         ///     be used.
         ///     The supported protocols in URI could either be <c>bolt</c> or <c>bolt+routing</c>.
-        ///     The protocol <c>bolt</c> should be used when creating a driver connecting to the Neo4j instance directly.
+        ///     The protocol <c>bolt</c> should be used when creating a driver connectiong to the Neo4j instance directly.
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver/V1/GraphDatabase.cs]: Same for this file `connecting` was correct.

File: Neo4j.Driver/Neo4j.Driver/V1/IAuthToken.cs [hunk 8]
```diff
@@ -33,7 +33,7 @@ public interface IAuthToken
 
     /// <summary>
     ///     This provides methods to create <see cref="IAuthToken"/>s for various authentication schemes supported by this driver. 
-    ///     The scheme used must be also supported by the Neo4j instance you are connecting to.
+    ///     The scheme used must be also supported by the Neo4j instance you are connectiong to.
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver/V1/IAuthToken.cs]: Same for this file. The `connecting` was correct.

**Review by zhenlineo — CHANGES_REQUESTED** [review]

**Comment by jsoref** [comment]
@zhenlineo: updated. Let me know if you're ok w/ this, and I'll squash...

**Comment by zhenlineo** [comment]
@jsoref Hi, could you squash the commits then we shall merge your PR?

**Comment by jsoref** [comment]
@zhenlineo: squashed

**Review by zhenlineo — APPROVED** [review]

**Pull request merged**